src/likelihoodratiotest.jl

```diff
@@ -0,0 +1,102 @@
+"""
+    LikelihoodRatioTest
+
+Results of MixedModels.likelihoodratiotest
+
+## Fields
+* `formulas`: Vector of model formulae
+* `models`: NamedTuple of the `dof` and `deviance` of the models
+* `tests`: NamedTuple of the sequential `dofdiff`, `deviancediff`, and resulting `pvalues`
+
+## Properties
+* `deviance`
+* `pvalues`
+
+"""
+struct LikelihoodRatioTest
+    formulas::AbstractVector{String}
+    models::NamedTuple{(:dof,:deviance)}
+    tests::NamedTuple{(:dofdiff,:deviancediff,:pvalues)}
+end
+
+Base.propertynames(lrt::LikelihoodRatioTest, private = false) = (
+    :deviance,
+    :formulas,
+    :models,
+    :pvalues,
+    :tests,
+)
+
+function Base.getproperty(lrt::LikelihoodRatioTest, s::Symbol)
+    if s == :dof
+        lrt.models.dof
+    elseif s == :deviance
+        lrt.models.deviance
+    elseif s == :pvalues
+        lrt.tests.pvalues
+    elseif s == :formulae
+        lrt.formulas
+    else
+        getfield(lrt, s)
```

Comment:
This should probably be `getproperty`.

Reply:
It has to be `getfield` or it's a bottomless recursion.

Reply:
(I found this out the hard way.)

Reply:
Yeah. I realized that as I was walking to the zoo... all the sudden, wait it's overloading `getproperty`... that would be bad. #StrollingThroughTheForestRealizations